src/main/resources/templates/login.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org" lang="en" >
+    <head>
+        <meta charset="UTF-8">
+        <title>Tic Tac Toe</title>
+        <link href="../static/css/style.css" th:href="@{/css/style.css}" rel="stylesheet" />
+    </head>
+    <body>
+      <div class="container">
+          <div class="menu-wrapper">
+              <ul class="menu">
+                  <li> pierwsza opcja </li>
+                  <li> druga opcja </li>
+                  <li> trzecia opcja </li>
+              </ul>
+          </div>
+          <div class="two-unit-container">
+              <div class="unit">
+                  <form>
+                     <ul class="left-unit-content">
+                         <li>
+                             <span class="input-reference">Login: </span>
+                             <input type="text" id="login" class="input-text" placeholder="login">
+                             <br><span class="input-warning"><!--warning--></span>
+                         </li>
+                         <li>
+                             <span class="input-reference">Password: </span>
+                             <input type="password" id="password1" class="input-password" placeholder="••••••••">
+                             <br><span class="input-warning"><!--warning--></span>
+                         </li>
+                         <li>
+                             <input type="submit" id="submit" class="submit" value="login">
+                         </li>
+                     </ul>
+                  </form>
+              </div>
+              <div class="separate-unit"></div>
+              <div class="unit">
+                  <ul class="right-unit-content">
+                      <li> info </li>
+                      <li> info </li>
+                      <li> info </li>
+                      <li> info </li>
+                      <li> info </li>
+                      <li> info </li>
+                      <li> info </li>
+                      <li> info </li>
+                      <li> info </li>
+                      <li> info </li>
+
+                  </ul>
+              </div>
+          </div>
+       </div>
+    </body>
+</html>
```

Comment:
Remember to add enter in the end of file https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file.